cmd/clusterawsadm/cloudformation/bootstrap/iam.go

```diff
@@ -71,6 +71,18 @@ func (t Template) policyFunctionMap() map[PolicyName]func() *iamv1.PolicyDocumen
 	}
 }
 
+func (t Template) GetPolicyDocs() error {
+	for _, name := range ManagedIAMPolicyNames {
+		policyDoc := t.GetPolicyDocFromPolicyName(name)
+		value, err := converters.IAMPolicyDocumentToJSON(*policyDoc)
+		if err != nil {
+			return err
+		}
+		fmt.Println(name, value)
+	}
+	return nil
+}
+
```

Comment:
I assume this is some kind of debug function, right? This should be removed or changed or something.

Reply:
Hey @Atharva-Shinde.

I don't see you changing this function. This function still doesn't make any sense. It's not returning anything, it doesn't set anything... And it's using `fmt.Println`, which should be forbidden by the linter anyways. :)